src/Components/CreateImageWizard/formComponents/ReviewStep.js

```diff
@@ -98,6 +98,7 @@ const ReviewStep = () => {
                         }
                     </List>
                 </Tab>
+                {releaseValues?.[getState()?.values?.release] === 'Red Hat Enterprise Linux (RHEL) 8' &&
```

Comment:
Nice start! I would recommend something more similar to line 118. 

`getState()?.values?.release.includes('rhel')`

This allows us to more generally check if we are building a rhel system or not.